studio/components/interfaces/Settings/Database/ConnectionPooling.tsx

```diff
@@ -140,7 +140,12 @@ export const PgbouncerConfig = ({ projectRef, bouncerInfo, connectionInfo }: Con
 
   const canUpdateConnectionPoolingConfiguration = useCheckPermissions(
     PermissionAction.UPDATE,
-    'projects'
+    'projects',
+    {
+      resource: {
+        project_id: projectRef,
```

Comment:
Shouldn't this be the projects id rather than the ref? Looking at other usages of useCheckPermissions it looks like we're passing the id (numeric) instead of the ref

Reply:
this should be the project id